src/components/visual-effects/background-parallax.tsx

```diff
@@ -11,6 +11,8 @@ export interface BackgroundParallaxProps extends HTMLAttributes<HTMLDivElement>
      *   the viewport, similar to `background-attachment: fixed`.
      */
     parallaxStrength?: number;
+    imageUrl?: string;
+    className?: string;
```

Comment:
You don't need `className`

Reply:
I don't want to add additional styles to the inline style:
```
.floatingCardBackground {
    width: 100vw;
    margin-left: -2%;
    padding: 7.8% 11.7%;
    aspect-ratio: 2;
}
```

Reply:
You don't need the className prop, this component already supports `className`.

interface BackgroundParallaxProps **`extends HTMLAttributes<HTMLDivElement>`**